types/ltx/index.d.ts

```diff
@@ -1,17 +1,17 @@
-// Type definitions for ltx 2.6
+// Type definitions for ltx 2.8
 // Project: github.com/node-xmpp/ltx/
 // Definitions by: PJakcson <https://github.com/PJakcson>
+//                 BendingBender <https://github.com/BendingBender>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
-// TypeScript Version: 2.1
 
-export {Element } from './lib/Element';
-export {nameEqual, attrsEqual, childrenEqual, equal} from './lib/equal';
-export {isNode, isElement, isText} from './lib/is';
-export {clone} from './lib/clone';
-export {createElement} from './lib/createElement';
-export {escapeXML, unescapeXML, escapeXMLText, unescapeXMLText} from './lib/escape';
-export {Parser} from './lib/Parser';
-export {parse} from './lib/parse';
-export {tag} from './lib/tag';
-export {tagString} from './lib/tagString';
-export {stringify} from './lib/stringify';
+export * from './lib/Element';
```

Comment:
Why re-export more symbols?

Reply:
It is actually quite useful to have all other types defined in Element, like ode and TextNode. I currently add types for all of @xmpp/* libs and these exports were already quite useful there.